hydroflow/src/lang/lattice/dom_pair.rs

```diff
@@ -0,0 +1,148 @@
+use std::cmp::Ordering;
```

Comment:
dom pair kind of sketches me out, can we add a big disclaimer that you have to be very careful in order for this to be a lattice?